src/components/common/ui/form/radio/RadioGroup.stories.tsx

```diff
@@ -0,0 +1,32 @@
+import type { Meta, StoryObj } from '@storybook/react';
+
+import RadioGroup from '@/components/common/ui/form/radio';
+/*
+const meta = {
+  title: 'Ui Kit/Components/form/radio/radioGroup',
+  component: RadioGroup,
+  parameters: {
+    layout: 'centered',
+  },
+  tags: ['autodocs'],
+} satisfies Meta<typeof RadioGroup>;
+
+export default meta;
+
+type Story = StoryObj<typeof meta>;
+export const Base: Story = {
+  args: {
+    options: [
+      { value: '0', label: 'Щомісячно' },
+      { value: '1', label: 'Щосеместрово' },
+      { value: '2', label: 'Щорічно' },
+    ],
+    sx: {
+      display: 'flex',
+      justifyContent: 'flex-start',
+      gap: '36px',
+    },
+    name: 'Виберіть спосіб оплати',
+  },
+};
+*/
```

Comment:
Uncomment this and separate formik version of RadioGroup from default one. Make sure everything works afterwards